code-challenges.test.js

```diff
@@ -15,35 +15,169 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+describe('shiftAndShuffle', () => {
+    const expected = ["yellow", "blue", "pink", "green"] || ["blue", "pink", "green", "yellow"] || ["pink", "green", "yellow", "blue"] || ["green", "yellow", "blue", "pink"];
+    const expected2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"] || ["aqumarine", "periwinkle", "indigo", "ochre", "saffron"] || ["periwinkle", "indigo", "ochre", "saffron", "aquarmaine"] || ["indigo", "ochre", "saffron", "aquarmaine", "periwinkle"] || ["ochre", "saffron", "aquarmaine", "periwinkle","indigo"]
+    it('removes the first item from the array and shuffles the remaining content', () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+      expect(shiftAndShuffle (colors1)).toEqual(expect.arrayContaining(expected));
+      expect(shiftAndShuffle (colors2)).toEqual(expect.arrayContaining(expected2))
+    });
+  })
+
```

Comment:
Hey Will - I understand why you wrote out all the possible combinations of the leftovers.  This should not be necessary when using arrayContaining.  You can just use one.  Good extra effort :)